OWNERS

```diff
@@ -0,0 +1,23 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- mnairn
+- maleck13
+- philbrookes
+- sedroche
+- aidenkeating
+- dimitraz
+- matskiv
+- JameelB
+- trepel
```

Comment:
@maleck13 @philbrookes I just created a list of approvers based on who had actively been working in this repo. It might be that you'd like to have this list smaller to limit who can approve PRs. A PR being approved will label it as ready to be automatically merged with tide (https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#automation-using-owners-files), assuming all other criteria for a merge are met (Tests passing etc..)